components/TARGET_PSA/inc/psa/client.h

```diff
@@ -37,7 +37,6 @@
 
 #define PSA_FRAMEWORK_VERSION   (0x0100) /**< Version of the PSA Framework API. */
 #define PSA_VERSION_NONE        (0L)     /**< Identifier for an unimplemented Root of Trust (RoT) Service. */
-#define PSA_SUCCESS             (0L) /**< A general result code for calls to psa_call()  indicating success.*/
```

Comment:
if you remove this add `#include psa/errors.h`

Reply:
i did not observe any compilation warning - but added anyway